### DISCUSSION TITLE
Telemetry issue

### DISCUSSION THREAD

**Discussion by t0sma86** [capsule-corp-ternoa/ternoa-hub#29] [General]
Hi, 

Since yesterday, I have an issue in Telemetry. Block propagation time is high 
![unknown](https://user-images.githubusercontent.com/108297916/192271506-be671d2a-1fc1-4a49-85d9-accaa2b7ec2c.png)
Multiple line displayed for nodes. 
![unknown-1](https://user-images.githubusercontent.com/108297916/192271670-9fce5932-30cf-47cc-93d6-8a5098ee681c.png)

Someone have the same issue ? Or it's only from my side? 

Thanks

**Comment by Herrlalwani** [comment]
@markopoloparadox @Leouarz

**Comment by ipapandinas** [comment]
From my side it is all good.

**Reply by t0sma86** [reply]
Did you chech the block propagation time ? Sort it by descending. Some nodes are really high since yesterday.
